.eslintrc.cli.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
```

Comment:
I think we can merge the 2 `.eslintrc.*.js` files by adding this: 
```json
 settings: {
    'import/resolver': {
      webpack: {
        config: './internals/webpack/webpack.dev.babel.js',
      },
      node: {
        extensions: ['.js', '.jsx', '.react.js'],
        moduleDirectory: ['node_modules', './src'],
      },	      
    },
```

it's cleaner and easyer to read

Reply:
We can't, because the CLI only needs the Webpack config and VS Code only needs the `node` config. Putting in both in the same file will still give `import/no-unresolved` warnings in VS Code.